dev-qt/qdbus/qdbus-5.15.0_beta4.ebuild

```diff
@@ -19,7 +19,3 @@ DEPEND="
 	~dev-qt/qtxml-${PV}
 "
 RDEPEND="${DEPEND}"
-
-QT5_TARGET_SUBDIRS=(
-	src/qdbus/qdbus
-)
```

Comment:
removing this won't end up building both qdbus and qdbusviewer?

Reply:
right, I was too fast there